doc/sf/jacobi_theta.qbk

```diff
@@ -0,0 +1,349 @@
+
+  jacobi_theta1(x, exp(-a));
+
+A more accurate computation will take advantage of [tau]:
+
+  jacobi_theta1tau(x, a / boost::math::constants::pi<T>());
+
+Internally, Boost implements the /q/ parameterization by taking the logarithm of /q/ and passing it to the [tau] parameterization; as such, using the [tau] parameterization directly will generally yield greater precision. As another example, if the complement of /q/ is known with great accuracy, then instead of:
+
+  jacobi_theta1(x, 1-q_complement);
+
+It is more accurate to use `__log1p` and pass in the result to the [tau] version:
+
+  jacobi_theta1tau(x, -boost::math::log1p(-q_complement) / boost::math::constants::pi<T>());
+
+Additional "minus 1" versions of the third and fourth theta functions are provided. Similar in spirit to `__expm1`, these functions return one less than the evaluated function, and yield increased accuracy when /q/ is small.
+
+[heading Testing]
+
+Results of the theta functions are tested against Wolfram Alpha data, as well as random values computed at high precision. In addition, the tests verify the majority of the identities described in [@https://dlmf.nist.gov/20.7 DLMF Chapter 20.7].
+
+[endsect] [/section:jacobi_theta_overview Overview of the Jacobi Theta Functions]
+
+[section:jacobi_theta1 Jacobi Theta Function [theta][sub 1]]
+
+[heading Synopsis]
+
+``
+  #include <boost/math/special_functions/jacobi_theta.hpp>
+``
+
+  namespace boost { namespace math {
+      template <class T>
+      ``__sf_result`` jacobi_theta1(T x, T q);
+
+      template <class T, class Policy>
+      ``__sf_result`` jacobi_theta1(T x, T q, const Policy&);
+
+      template <class T>
+      ``__sf_result`` jacobi_theta1tau(T x, T tau);
+
+      template <class T, class Policy>
+      ``__sf_result`` jacobi_theta1tau(T x, T tau, const Policy&);
+  }} // namespaces
+
+[heading Description]
+
+The functions calculate the value of first [link math_toolkit.jacobi_theta.jacobi_theta_overview Jacobi Theta function], parameterized either in terms of the nome /q/:
+
+[equation jacobi_theta1] [/ \theta_1(x, q) = 2 \sum_{n=0}^\infty (-1)^n q^{(n+\frac{1}{2})^2} \sin((2n+1)x) ]
+
+Or in terms of an imaginary [tau]:
+
+[equation jacobi_theta1tau] [/ \theta_1(x|\tau) = 2 \sum_{n=0}^\infty (-1)^n \exp(i\pi\tau{(n+0.5)^2}) \sin((2n+1)x) ]
+
+The nome /q/ is restricted to the domain (0, 1), returning the result of __domain_error otherwise. The following graph shows the theta function at various values of /q/:
+
+[graph jacobi_theta1]
+
+[optional_policy]
+
+[heading Accuracy]
+
+The following [link math_toolkit.ulps_plots ULPs plot] is representative, fixing /q/=0.5 and varying /x/ from 0 to 2[pi]:
+
+[graph jacobi_theta1_float]
+
+The envelope represents the function's [link math_toolkit.cond condition number]. Note that relative accuracy degenerates periodically near [theta][sub 1]=0.
```

Comment:
A better link would be:

https://en.wikipedia.org/wiki/Condition_number

(I couldn't get the shard to the one variable section, but that would be even better.